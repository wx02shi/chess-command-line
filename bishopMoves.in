game human human
setup
+ K e1 
+ B c1
+ P e2
= black
+ K e8
done
move c1 e3
move e8 f8
move e3 d2
move f8 e8
move d2 c3
move e8 f8
move c3 a1
move f8 e8
move e2 e4
move e8 f8
move e4 e5
move f8 e8
move a1 f5
