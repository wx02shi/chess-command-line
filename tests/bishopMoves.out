8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ B K _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _P_ _
1 _ B K _ 

  abcdefgh
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _P_ _
1 _ B K _ 

  abcdefgh
Setup completed!
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ B _ 
2  _ _P_ _
1 _ _ K _ 

  abcdefgh
8  _ _ k _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ B _ 
2  _ _P_ _
1 _ _ K _ 

  abcdefgh
8  _ _ k _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ BP_ _
1 _ _ K _ 

  abcdefgh
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ BP_ _
1 _ _ K _ 

  abcdefgh
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ B _ _ 
2  _ _P_ _
1 _ _ K _ 

  abcdefgh
8  _ _ k _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ B _ _ 
2  _ _P_ _
1 _ _ K _ 

  abcdefgh
8  _ _ k _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _P_ _
1 B _ K _ 

  abcdefgh
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _P_ _
1 B _ K _ 

  abcdefgh
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2  _ _ _ _
1 B _ K _ 

  abcdefgh
8  _ _ k _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2  _ _ _ _
1 B _ K _ 

  abcdefgh
8  _ _ k _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ P _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 B _ K _ 

  abcdefgh
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ P _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 B _ K _ 

  abcdefgh
Invalid move
Final Score:
White: 0
Black: 0
